packages/assets-controllers/src/TokensController.ts

```diff
@@ -344,8 +347,7 @@ export class TokensController extends BaseController<
    *
    * @param tokensToImport - Array of tokens to import.
    */
-  async addTokens(tokensToImport: Token[]) {
-    const releaseLock = await this.mutex.acquire();
```

Comment:
Looks like we are removing the mutex lock here. Is that intentional? If so, what's the rationale?

Reply:
No, not intentional sorry! It missed me somehow

Reply:
Added again the mutex logic